internal/kibana/packages.go

```diff
@@ -27,9 +28,32 @@ func (c *Client) InstallPackage(pkg packages.PackageManifest) ([]packages.Asset,
 	return processResults("install", statusCode, respBody)
 }
 
+// InstallZipPackage installs the local zip package in Fleet.
+func (c *Client) InstallZipPackage(zipFile string) ([]packages.Asset, error) {
+	path := fmt.Sprintf("%s/epm/packages", FleetAPI)
+
+	fileContents, err := os.ReadFile(zipFile)
+	if err != nil {
+		return nil, errors.Wrap(err, "failed to read zip file")
+	}
+
+	req, err := c.newRequest(http.MethodPost, path, fileContents)
```

Comment:
It'd be probably better for `newRequest` to receive a reader as body. With `ReadFile` we have to put the whole package in memory. We don't have so big packages now, but we might have soon.
```suggestion
	body, err := os.Open(zipFile)
	if err != nil {
		return nil, errors.Wrap(err, "failed to read zip file")
	}
	defer body.Close() 

	req, err := c.newRequest(http.MethodPost, path, body)
```

`sendRequest` could keep its interface if we don't want to modify the rest of methods, by wrapping the body with a `bytes.NewReader`.
```
func (c *Client) sendRequest(method, resourcePath string, body []byte) (int, []byte, error) {
	request, err := c.newRequest(method, resourcePath, bytes.NewReader(body))
	...
```